### DISCUSSION TITLE
Keyboard events are sent wrong （down hold released）

### DISCUSSION THREAD

**Discussion by Likkkez** [rustdesk/rustdesk#10385] [General]
### Bug Description

Whenever I hold down a key it just instantly releases on the host machine rather than being held down.

### How to Reproduce

https://www.keyboardtester.com/tester.html open keyboard tester and try holding down a key

### Expected Behavior

Expected it to release the key when I let go of the button

### Operating system(s) on local (controlling) side and remote (controlled) side

linux

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.3.6

### Screenshots

no

### Additional Context

This sucks because I want to work in graphical applications and not just typing stuff. It should actually recognize the key events properly instead of this problem.

**Comment by rustdesk** [comment]
We knew this and this is intended. We have not decided to do the change it yet.

# BTW, we totally understand your concern.

**Reply by Likkkez** [reply]
> We knew this and this is intended. We have not decided to do the change it yet.
> 
> # BTW, we totally understand your concern.

It would be great to have a checkbox perhaps. I wanted to use Unreal Engine but as soon as I try to move the camera or do anything the character immediately stops because the keys are sent that way. It's a real shame because the response time is perfect and near instant its just that limitation that stops me from doing any 3D stuff.

**Comment by Plyrolith** [comment]
Same problem here. Our company (3D) animation is currently evaluating RustDesk Pro and this is a major concern.

**Comment by Misha-133** [comment]
Plus 1 on this. We were really excited when we switched to a self hosted rustdesk pro server, the latency & quality is amazing. Until we stumbled into this, and this turns out to be a real deal breaker. 
really wish to see this fixed, as otherwise we'd have to switch away from rustdesk.

**Comment by cattrooper28** [comment]
@rustdesk Will this bug ever be fixed?

**Comment by ckeylis** [comment]
I have noted this as well recently in #14362.
